src/ReactDOMTestUtils.re

```diff
@@ -84,7 +84,7 @@ module Simulate = {
 
 external document: Dom.document = "document";
 
-[@mel.send]
+[@mel.return nullable] [@mel.send]
```

Comment:
for my understanding: we're adding `@mel.return nullable` because `querySelector` may return `null` or `undefined` and we want to convert it to an option?

Reply:
Actually it can only return `null`. https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector

Maybe I should use `null_to_opt`? To be more specific.

Reply:
this might be a good idea, but no strong opinions